.env.example

```diff
@@ -25,5 +25,6 @@ MAIL_ENCRYPTION=null
 NORTHSTAR_URI=http://northstar.dev:8000
 NORTHSTAR_API_KEY=abc4324
 
-PHOENIX_URI=https://dev.dosomething.org:8888
+PHOENIX_URI=https://staging.beta.dosomething.org
+PHOENIX_API_URI=https://staging.beta.dosomething.org/api
```

Comment:
@sbsmith86 I updated this in my [last PR](https://github.com/DoSomething/gladiator/pull/120/files#diff-7fdf976827c0f65f17c003adf2ba56d9R16) so the api thing should work without adding a second var for this.